src/weatherAPI/Weather.java

```diff
@@ -12,13 +12,22 @@
  */
 
 public class Weather {
-    static double currentTemp;          // Current temperature in ºF
-    static String currentCondition;     // Current weather condition
+    public static double currentTemp;          // Current temperature in ºF
+    public static String currentCondition;     // Current weather condition
     // API Key for OpenWeatherMap OneCall 3.0 API
     private static final String API_KEY = "3a86deaa95c6eea2a6462d1d879afc68";
     // API Endpoint
     private static final String API_ENDPOINT = "https://api.openweathermap.org/data/3.0/onecall?lat=%s&lon=%s&exclude=minutely,hourly,daily,alerts&appid=%s&units=imperial";
 
+    /**
+     * Overridden method to accept a String location and call Geocoder directly
+     * @param location
+     */
+    public static void getWeather(String location) {
```

Comment:
Wrote this so we can just put the zip code/city directly into the getWeather method, and then it does the Geolocator stuff behind the scenes and still uses the original getWeather code to do the URL and JSON stuff